packages/cozy-stack-client/src/FileCollection.spec.js

```diff
@@ -331,6 +331,43 @@ describe('FileCollection', () => {
         }
       })
     })
+
+    it('should update a file with metadata', async () => {
+      const metadataId = '2460a631-ae55'
+      client.fetchJSON.mockReturnValueOnce({
+        data: {
+          id: metadataId
+        }
+      })
+      const data = new File([''], 'mydoc.odt')
+      const params = {
+        fileId: '59140416-b95f',
+        checksum: 'a6dabd99832b270468e254814df2ed20',
+        metadata: { type: 'bill' }
```

Comment:
[Here](https://github.com/cozy/cozy-doctypes/blob/a83a83df999fd64c7246308120f9928430aed6b4/docs/README.md}, it's written that the metadata key is `cozyMetadata`. Are files a special case here due to history ?

Reply:
Files are special anyway, this is why we have a FileCollection. But Files also have a new "metadata" attributes (https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.files.md#files) to store metadata not related to cozy. For instance we can store the type of the file (not PDF or PNG, but its business type like 'bill' or 'identity document' or 'payslip'...). 

Just saw that the PR is not merged yet : https://github.com/cozy/cozy-doctypes/pull/101

Reply:
We had discussions about how to store and index times. This should have no impact on your PR